src/main/java/uk/ac/ebi/biostudies/api/util/SnapshotAwareDirectoryTaxonomyWriter.java

```diff
@@ -0,0 +1,41 @@
+package uk.ac.ebi.biostudies.api.util;
+
+import org.apache.lucene.facet.taxonomy.directory.DirectoryTaxonomyWriter;
+import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.index.KeepOnlyLastCommitDeletionPolicy;
+import org.apache.lucene.index.SnapshotDeletionPolicy;
+import org.apache.lucene.store.AlreadyClosedException;
+import org.apache.lucene.store.Directory;
+
+import java.io.IOException;
+
+public class SnapshotAwareDirectoryTaxonomyWriter extends DirectoryTaxonomyWriter {
+
+    private SnapshotDeletionPolicy facetIndexSnapShot;
+
+    public SnapshotAwareDirectoryTaxonomyWriter(Directory facetDirectory, IndexWriterConfig.OpenMode openMode) throws IOException{
+            super(facetDirectory, openMode);
+    }
+
+    @Override
+    protected IndexWriterConfig	createIndexWriterConfig(IndexWriterConfig.OpenMode openMode){
```

Comment:
Extra space between return type and method name. Let's start formatting the code to the default IntelliJ style

Reply:
This happened during merge ...